kinto/core/statsd.py

```diff
@@ -6,13 +6,18 @@
 except ImportError:  # pragma: no cover
     statsd_module = None
 
+from pyramid.exceptions import ConfigurationError
 from six.moves.urllib import parse as urlparse
 
 from kinto.core import utils
 
 
 class Client(object):
     def __init__(self, host, port, prefix):
+        if statsd_module is None:
```

Comment:
Should this only be raised if statsd is configured, as explained in [the docs above](https://github.com/Kinto/kinto/pull/832/files#diff-de0a9b67ffe2874a076b5fdb15bad484R24)?